### PR TITLE
Grouped products work with Single Stock

### PR DESCRIPTION
**Pull request by nmalevanec**
### Description
Grouped products can only be assigned to default source

### Fixed Issues (if relevant)
1. magento-engcom/msi#684: Grouped products work with Single Stock

### CODE REVIEW

File: app/code/Magento/InventoryGroupedProduct/etc/module.xml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+<!--
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+-->
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:Module/etc/module.xsd">
+    <module name="Magento_InventoryGroupedProduct" setup_version="1.0.0" >
+        <sequence>
+            <module name="Magento_InventoryConfiguration" />
```
Comment (naydav) [inline app/code/Magento/InventoryGroupedProduct/etc/module.xml]: We don't need sequence on `Magento_InventoryConfiguration`

Comment (nmalevanec) [reply app/code/Magento/InventoryGroupedProduct/etc/module.xml]: Removed.

File: app/code/Magento/InventorySales/Plugin/StockState/CheckQuoteItemQtyPlugin.php [hunk 2]
```diff
@@ -99,25 +94,26 @@ public function aroundCheckQuoteItemQty(
...
-
-        $isSalableResult = $this->isProductSalableForRequestedQty->execute($productSku, (int)$stockId, $qty);
-
-        if ($isSalableResult->isSalable() === false) {
-            /** @var ProductSalabilityError $error */
-            foreach ($isSalableResult->getErrors() as $error) {
-                $result->setHasError(true)->setMessage($error->getMessage())->setQuoteMessage($error->getMessage())
-                       ->setQuoteMessageIndex('qty');
+        $stockId = $this->getStockIdForCurrentWebsite->execute();
+        if ($this->defaultStockProvider->getId() === $stockId) {
```
Comment (naydav) [inline app/code/Magento/InventorySales/Plugin/StockState/CheckQuoteItemQtyPlugin.php]: You can decrease complexity via
```
if ($this->defaultStockProvider->getId() !== $stockId) {
    return $proceed($productId, $itemQty, $qtyToCheck, $origQty, $scopeId);
}
...
```

Comment (nmalevanec) [reply app/code/Magento/InventorySales/Plugin/StockState/CheckQuoteItemQtyPlugin.php]: Done.

**Pull request merged**